modules/navier_stokes/examples/flow-over-circle/tests

```diff
@@ -1,28 +1,19 @@
 [Tests]
   issues = '#21133 #29150'
   design = 'insfv.md linear_fv_design.md'
-  [vortex_shedding]
+  [vortex_shedding-nonlinear-physics]
     type = 'CSVDiff'
-    input = 'header.i mesh.i flow_over_circle.i executioner_postprocessor.i'
-    csvdiff = 'flow_over_circle_out.csv'
-    requirement = 'The system shall be able to solve transient flow resulting in vortex shedding in 2D using the finite volume method'
-    cli_args = 'refinement=2 Executioner/end_time=2e-3 Outputs/file_base=flow_over_circle_out'
+    input = 'flow_over_circle-physics.i'
+    csvdiff = 'flow_over_circle-physics_out.csv'
+    requirement = 'The system shall be able to solve transient flow resulting in vortex shedding in 2D using the Navier Stokes Flow Physics syntax'
```

Comment:
This is implementation specific. Shouldn't have capitalized concepts like "Flow Physics"